src/backend/distributed/deparser/citus_ruleutils.c

```diff
@@ -1345,6 +1345,16 @@ convert_aclright_to_string(int aclright)
 			return "TEMPORARY";
 		case ACL_CONNECT:
 			return "CONNECT";
+#if PG_VERSION_NUM >= PG_VERSION_15
```

Comment:
hmm, why do we need to change for PG15? Did we miss that in PG 15 support?

Reply:
Yes exactly, we missed it in PG15 support

Reply:
We can add it later, creating an issue for it to remember this. For now, removing from this PR.